resources/RemoveInactiveUsersRunner.php

```diff
@@ -57,36 +71,60 @@
     if ($lastLoginDate) { // null lastLoginDate check
         $interval = $today->diff($lastLoginDate);
     } else { // This might only be run once, since new users always have field filled.
-        echo "User has no lastLoginDate (it may have been a very long time.)\n";
-        echo "Deleting user.\n";
+        echo "Deleting this user as it has no last login date " .
+        "(it may have been a very long time). \n";
+        if ($dryRun == true) {  // If dry run option has been selected
+             //writing to file to say who would have been deleted
+            file_put_contents("InactiveUsersToBeDeleted.txt", $user->getId() . ", `null`, " .
+            "`?`\n", FILE_APPEND);
+        };  // no else needed.
         deleteUser($user, $entityManager);
```

Comment:
`deleteUser` is called here regardless of whether `$dryRun == true` is `true` or not. The method call should be in an `else` clause.